bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -115,23 +116,26 @@ private void initiateConnexion() {
 
     public @Nullable OpenUVResult getUVData(String latitude, String longitude, String altitude) {
         try {
-            String jsonData = HttpUtil.executeUrl("GET", String.format(QUERY_URL, latitude, longitude, altitude),
-                    header, null, null, REQUEST_TIMEOUT_MS);
-            OpenUVResponse uvResponse = gson.fromJson(jsonData, OpenUVResponse.class);
-            if (uvResponse != null) {
-                String error = uvResponse.getError();
-                if (error == null) {
-                    updateStatus(ThingStatus.ONLINE);
-                    return uvResponse.getResult();
+            String url = String.format(QUERY_URL, latitude, longitude, altitude);
+            String jsonData = HttpUtil.executeUrl("GET", url, header, null, null, REQUEST_TIMEOUT_MS);
+            try {
+                OpenUVResponse uvResponse = gson.fromJson(jsonData, OpenUVResponse.class);
+                if (uvResponse != null) {
+                    String error = uvResponse.getError();
+                    if (error == null) {
+                        updateStatus(ThingStatus.ONLINE);
+                        return uvResponse.getResult();
+                    }
+                    throw new OpenUVException(error);
                 }
-                throw new OpenUVException(error);
+            } catch (JsonSyntaxException e) {
+                logger.info("No json received when calling `{}` : {}", url, jsonData);
```

Comment:
Should thing be marked as **OFFLINE** in this scenario?

Reply:
I think the message shoud rather be "No **valid** json received ..."
And a DEBUG log should be sufficient ?

Reply:
I would not mark it as OFFLINE because invalid json can happen from time to time (reason of this PR). Fine to change it to debug.